irg_gazebo_plugins/src/IRGCameraSimPlugins/README.md

```diff
@@ -61,10 +61,10 @@ settled on a VisualPlugin instead.
 ### Parameters
 The plugins can be initialized with user-defined values using the following parameters:
  - `<topic_uid>` - Changes topic from `/gazebo/plugins/camera_sim/<topic>` to `/gazebo/plugins/camera_sim/<topic_uid>/<topic>`.
- - `<exposure>` - Multiply original image by this value. Default = 1.0
- - `<energy_conversion>` - Pixel value per lux-second. A conversion factor from luminous energy to normalized sensor output. Default = 1.0
- - `<read_noise>` - Read noise coefficient. Default = 0.64
- - `<shot_noise>` - Shot noise coefficient. Default = 0.09
+ - `<exposure>` - Exposure time (seconds). Default = 1.0
+ - `<energy_conversion>` - Pixel value in range {0.0, 1.0} per lux-second. A conversion factor from luminous energy to normalized sensor output. Default = 1.0
+ - `<read_noise_std_dev>` - Read noise standard deviation (in integer pixel value where pixel values are in the range {0, 2<sup>adc_bits</sup> - 1}). Default = 0.8
+ - `<shot_noise_coeff>` - Shot noise coefficient (unitless). Default = 0.3
```

Comment:
See my comment on range notation in the .frag file.  I think it applies here as well.